src/content/docs/guides/how-biome-works.mdx

```diff
@@ -335,19 +335,26 @@ This is a current limitation that we will fix in a future release.
 
 ### Glob syntax explained
 
+A glob pattern specifies a set of filenames.
+Biome supports the following globs:
 
-The syntax and meaning of these two options loosely follow the [globset rules](https://docs.rs/globset/latest/globset/#syntax) but without the ability to set options.
+- `*` matches zero or more characters. It cannot match the path separator `/`.
+- `**` recursively matches directories and files.
+- `?` matches any character. It cannot match the path separator `/`.
+- `[ab]` matches any character among `a` and `b`.
+- `[a-z]` matches any character between `a` and `z`.
+- `[^a-z]` matches any character which is not between `a` and `z`.
```

Comment:
I don't think we support this pattern?
https://github.com/biomejs/biome/blob/a51ef9d771c526c5df42da0167a65813b7e80854/crates/biome_service/src/matcher/pattern.rs#L56-L57

Maybe we should use the docs here: https://github.com/biomejs/biome/blob/a51ef9d771c526c5df42da0167a65813b7e80854/crates/biome_service/src/matcher/pattern.rs#L40-L64

Reply:
Thanks!